client/signup/steps/intent/index.tsx

```diff
@@ -78,7 +83,14 @@ export default function IntentStep( props: Props ): React.ReactNode {
 			headerImageUrl={ intentImageUrl }
 			subHeaderText={ subHeaderText }
 			fallbackSubHeaderText={ subHeaderText }
-			stepContent={ <IntentScreen canImport={ canImport } onSelect={ submitIntent } /> }
+			stepContent={
+				<IntentScreen
+					intents={ intents }
+					intentsAlt={ intentsAlt }
+					onSelect={ submitIntent }
+					preventWidows={ preventWidows }
```

Comment:
Added this as asked in the issue "call said dep from within Calypso and pass the result as a prop to the package".